ldcbench.http-node/pom.xml

```diff
@@ -46,6 +46,31 @@
       <artifactId>spring-webmvc</artifactId>
     </dependency>
     <!-- ~~~~~~~~~~~~~~~~~~~ End Spring ~~~~~~~~~~~~~~~~~~~~~~ -->
+    <!-- ~~~~~~~~~~~~~~~~~~~ HDT ~~~~~~~~~~~~~~~~~~~~~~ -->
+            <!-- https://mvnrepository.com/artifact/org.rdfhdt/hdt-java-package -->
+            <dependency>
+                <groupId>org.rdfhdt</groupId>
+                <artifactId>hdt-java-package</artifactId>
+                <version>2.1.2</version>
+                <type>pom</type>
+                <exclusions>
+                    <exclusion>
+                        <groupId>junit</groupId>
+                        <artifactId>junit-dep</artifactId>
+                    </exclusion>
+                </exclusions>
+            </dependency>
+            <dependency>
+                <groupId>org.rdfhdt</groupId>
+                <artifactId>hdt-jena</artifactId>
+                <version>2.1.2</version>
+            </dependency>
+            <dependency>
+                <groupId>org.rdfhdt</groupId>
+                <artifactId>hdt-api</artifactId>
+                <version>2.1.2</version>
+            </dependency>
+            <!-- ~~~~~~~~~~~~~~~~~~~ HDT END ~~~~~~~~~~~~~~~~~~~~~~ -->
```

Comment:
The indentation is not correct. Please fix that since otherwise somebody will fix it later with a completely unrelated commit.